Classes depending on ASP.NET Core.dgml

```diff
@@ -0,0 +1,2099 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DirectedGraph DataVirtualized="True" Layout="Sugiyama" ZoomLevel="-1" xmlns="http://schemas.microsoft.com/vs/2009/dgml">
```

Comment:
I recommend against using this deprecated tool, which doesn't even work on recent VS. Consider using https://app.diagrams.net/ for diagrams - it generates a model file and pictures, and you can include both in your source.

Reply:
@bgavrilMS : this deprecated tool generates a graph from code. For a graph if this size this would have taken me 1 week to get this precision (think of listing all the classes and their dependencies, all that without making mistakes).